cloud/amazon/ec2_ami.py

```diff
@@ -44,7 +44,7 @@
     default: 300
   state:
     description:
-      - create or deregister/delete image
+      - create (present), register (register), deregister/delete image (absent)
```

Comment:
instead of another state, this should just be an alternate way for present to work, or at the very least we would need registered and unregistered...